### PR TITLE
Removed compatibility with shield [PC-1079]

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- The X8 is not compatible with the Portenta Vision Shield, so I removed this from the compatibility section.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/compatibility.yml [hunk 1]
```diff
@@ -2,8 +2,6 @@ software:
   - arduino-ide
   - arduino-cli
 hardware:
-  shields:
-    - portenta-vision-shield
   boards:
     - portenta-breakout
```
Comment (martab1994) [inline content/hardware/04.pro/boards/portenta-x8/compatibility.yml:6]: Thanks @jhansson-ard! Can we also move Portenta Breakout under Carriers section? Then it's perfect!

**Review by martab1994 — CHANGES_REQUESTED** [review]

File: content/hardware/04.pro/boards/portenta-x8/compatibility.yml [hunk 2]
```diff
@@ -2,8 +2,6 @@ software:
   - arduino-ide
   - arduino-cli
 hardware:
-  shields:
-    - portenta-vision-shield
   boards:
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-x8/compatibility.yml:5]:
```suggestion
```

File: content/hardware/04.pro/boards/portenta-x8/compatibility.yml (hunk 1 above)
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-x8/compatibility.yml:6]:
```suggestion
```

**Comment by jhansson-ard** [comment]
I removed it now @martab1994 ⭐

**Review by martab1994 — APPROVED** [review]
Perfect thanks!

**Pull request merged**